### DISCUSSION TITLE
Error when using Appwrite CLI to Init a Function

### DISCUSSION THREAD

**Discussion by MoeRayo** [appwrite/appwrite#5030] [Q&A] [answered]
Hello everyone, 
I have issues using the Appwrite CLI to init a function in windows.

I keep getting this error below:
```js
✗ Error Command failed: cmd /c "git clone --depth 1 --sparse https://github.com/appwrite/functions-starter ."
'cmd' is not recognized as an internal or external command,
operable program or batch file.
 

Suggestion: It appears that git is not installed, try installing git then trying to run this command again.
```
NB: I have git installed

I followed the suggestions in this GitHub [issue](https://github.com/appwrite/appwrite/discussions/4439) and upgraded my Git version to ` 2.39.1,` but it is still the name

🎲 Appwrite version
Version 10.2.0

💻 Operating system
Windows

🧱 Your Environment

Windows: 11
Node Version: 18.13.0
Git Version 2.39.1
Appwrite CLI version: 1.1.1

**Comment by stnguyen90** [comment, marked as answer]
> 'cmd' is not recognized as an internal or external command

This seems to indicate there is something wrong with your PATH. Maybe [this](https://stackoverflow.com/questions/32035577/cmd-is-not-recognized-as-an-internal-or-external-command-operable-program-or) will help.
